aries/aries_vcx/src/handlers/out_of_band/receiver.rs

```diff
@@ -38,6 +41,25 @@ impl OutOfBandReceiver {
         }
     }
 
+    pub fn from_json_string(oob_json: &str) -> VcxResult<Self> {
+        Ok(Self {
+            oob: from_json_string(oob_json)?,
+        })
+    }
+
+    pub fn from_base64_url(base64_url_encoded_oob: &str) -> VcxResult<Self> {
```

Comment:
same nit about naming; wasn't immediately sure what the input of this function was expected to be. i.e. something like `from_base64_url_encoded_oob` might be more specific.

also general question; will this API get much usage from ppl? i imagine most ppl are interested in using a "from invitation URL" or "from invitation message" API.

Reply:
> also general question; will this API get much usage from ppl? i imagine most ppl are interested in using a "from invitation URL" or "from invitation message" API.

I don't necessarily expect so, the most common being from URL / Invitation JSON. The first example I can think of now is if someone does like a custom deep linking implementation (so it being closed source/specific to their org/app), which could be useful to parse the base64url from. But in that same example they could just implement the base64url parsing, so it's kinda mute.

Would your thought here be to remove it from the public API, making it private for usage only by the other methods, for simplicity sake?

Reply:
ye i think that'd be my preference